packages/gatsby-theme-newrelic/src/utils/createTessen.js

```diff
@@ -40,7 +44,7 @@ const canSendAction = ({ config, name, category }) =>
 
 const tessenAction =
   (action, config) =>
-  ({ eventName, category, ...properties }) => {
+  async ({ eventName, category, ...properties }) => {
```

Comment:
The test modifications you needed to add, is it due to us making this `async`? reason i ask is i'm wondering if we need to do some promise handling anywhere else in the various repos that would use this 🤷